include/a3/data_f/$PBOPREFIX$

```diff
@@ -0,0 +1 @@
+a3\data_f
```

Comment:
Why are those necessary? Those files are already in folders representing prefix, and armake doesn't need them.

Reply:
Because some tools (compiler) search for pboprefix to be sure that it's what it should be.
Otherwise the prefix might aswell be include\a3\data_f or ace\include\a3\data_f or whateve

Reply:
EOF?

Reply:
Can be present, any normal tool will parse it just fine.

Reply:
Some bad tools just take the whole $PBOPREFIX$ as prefix.
So if you pack with these you end up with prefix\n 
soo stupid.. 
There is no reason to have newline at the end here really anyway.

Reply:
Forget those tools, just leave it as is. ;)

Reply:
cpbo is one of them. And I use that to build pbo's to quickly test things :D

Reply:
Judging.

Reply:
Solution would be to write a script that let's you add armake pbo packing to context menu.
Context menu stuff is a chore to do though.